doc/account.md

```diff
@@ -1,8 +1,7 @@
 Account
 =======
 
-Configuration
--------------
+## Configuration
```

Comment:
I started having to need more sub headings, but I can use the underline for main headings and ### for sub headings under that since ---- is the same as ##.